blog/_posts/2018-06-20-scala-pride.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: blog-detail
+post-type: blog
+by: Heather Miller
+title: "Scala ❤️ Pride 2018"
+---
+
+It's that time of year! Pride 2018. In honor of all of us Scalamaris that are advocates of or identify with the LGTB community–we have a little bit of Scala love for you. I've created special Pride Scala swirls for just about every pride flag I could think of. And perhaps the best part! They're all available as t-shirts, available on [Teespring](http://teespring.com). 100% of proceeds for t-shirt sales are donated to [the Trevor Project](https://www.thetrevorproject.org/). (Go check it out!)
```

Comment:
LGBT (instead of LGTB)?

Reply:
Also “community-we have” sounds weird. Maybe you mean “community. We have”?

Reply:
Goodness. SO MANY TYPOS. Thank you @julienrf!

Reply:
In “community-we have”, the dash is actually an em dash. http://www.thepunctuationguide.com/em-dash.html